source/specification_language_description.rst

```diff
@@ -283,38 +283,38 @@ describing the group. The ``doc`` key is required.
 
 .. _sec-data-type:
 
-``data_type_inc`` and ``data_type_def``
+``{{ data_type_inc }}`` and ``{{ data_type_def }}``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The concept of a data_type is similar to the concept of Class in object-oriented programming.
 A data_type is a unique identifier for a specific type of group (or dataset) in a specfication.
 By assigning a data_type to a group (or dataset) enables others to reuse that type by inclusion or
 inheritance (*Note:* only groups (or datasets) with a specified type can be reused).
 
-- ```data_type_def```: This key is used to define (i.e, create) a new data_type and to assign that type to
+- ```{{ data_type_def }}```: This key is used to define (i.e, create) a new data_type and to assign that type to
   the current group (or dataset).
 
-- ```data_type_inc```: The value of the ``data_type_inc`` key describes the base type
+- ```{{ data_type_inc }}```: The value of the ``{{ data_type_inc }}`` key describes the base type
   of a group (or dataset). The value must be an existing type.
 
-Both ```data_type_def``` and ```data_type_inc``` are optional keys.
+Both ```{{ data_type_def }}``` and ```{{ data_type_inc }}``` are optional keys.
 To enable the unique identification, every group (and dataset) must either have a fixed name and/or a
 unique data_type. This means, any group (or dataset) with a variable name must have a unique data_type.
 
-The data_type is determined by the value of the ``data_type_def`` key or if no new
-type is defined then the value of ``data_type_inc`` is used to determine type. Or in other
+The data_type is determined by the value of the ``{{ data_type_def }}`` key or if no new
+type is defined then the value of ``{{ data_type_inc }}`` is used to determine type. Or in other
 words, the data_type is determined by the last type in the ancestry (i.e, inheritance hierarchy) of an object.
 
 
-**Reusing existing data_types**
+**Reusing existing {{ data_types }}**
 
-The combination of ```data_type_inc``` and ```data_type_def``` provides an easy-to-use mechanism for
+The combination of ```{{ data_type_inc }}``` and ```{{ data_type_def }}``` provides an easy-to-use mechanism for
 reuse of type specifications via inheritance (i.e., merge and extension of specifications) and inclusion (i.e,
 embedding of an existing type as a component, such as a subgroup, of a new specification). Here an overview
 of all relevant cases:
 
 +------------------------+------------------------+------------------------------------------------------------------------+
-| ``data_type_inc`` | ``data_type_def`` |  Description                                                           |
+| ``{{ data_type_inc }}`` | ``{{ data_type_def }}`` |  Description                                                           |
```

Comment:
I think here we need to be careful as replacing with variable length strings may lead to misalignment in the table.

Reply:
Ugh rst is so annoying. I can fix this